automation/terraform/modules/kubernetes/testnet/helm.tf

```diff
@@ -237,12 +240,14 @@ resource "helm_release" "snark_workers" {
 # Archive Node
 
 resource "helm_release" "archive_node" {
+  provider   = helm.testnet_deploy
+
   count       = var.archive_node_count
   
-  name        = "archive-node-${count.index}"
+  name        = "archive-node-${count.index + 1}"
   repository  = local.use_local_charts ? "" : local.mina_helm_repo
   chart       = local.use_local_charts ? "../../../../helm/archive-node" : "archive-node"
-  version     = "0.3.3"
+  version     = "0.4.6"
```

Comment:
The other versions are 0.4.5 just want to make sure this is correct

Reply:
Yep, all good here - note: the versions across charts won't necessarily stay in sync